tests/test_ivf_index.cpp

```diff
@@ -12,6 +12,8 @@
 #include <cstring>
 #include <map>
 #include <random>
+#include <set>
+#include <algorithm>
```

Comment:
How about removing algorithm? I don't see where we use it

Reply:
It's used in a `std::find` call in this file.

Reply:
Hi @mfixman, I think you are right. I'm wondering if you could put #include<alogrithm> in between #include <omp.h> and #include <cstddef>. The format doesn't pass. Once you fix it, I could merge it